rosdep/base.yaml

```diff
@@ -5294,7 +5294,7 @@ libqt4-sql-psql:
   macports: [qt4-mac]
   nixos: [qt4]
   ubuntu: [libqt4-sql-psql]
-libqt5-charts:
+libqt5-charts-dev:
```

Comment:
I think it is a good idea to add in the `-dev` suffix; that way in the future we could add in a runtime library.

@satoshi-ota Given that you originally added this key in https://github.com/ros/rosdistro/pull/39705, are you OK if we rename it here?

Reply:
@clalancette No problem. Thank you :+1: